### PR TITLE
docs(storybook): L3-2028 added links to figma and storybook

### PR DESCRIPTION
**Pull request by constantinehuzenko**
**Jira ticket**

[L3-2028](https://phillipsauctions.atlassian.net/browse/L3-2028)

**Screenshots**
image
image
image

**Summary**

Added overview jsdoc comment with Figma link to all components.

**Change List (describe the changes made to the files)**

- Added storybook and figma link to all represented components in Figma

**Acceptance Test (how to verify the PR)**

- Hover over the component name.
- Click on the Figma link.
- Verify that the Figma design matches the component.
- Locate the Dev Resources section and click on the link.
- Verify that the new page matches the Figma page.

**Regression Test**

- (Optional) Add verification steps to make sure this PR doesn't break old functionality

**Evidence of testing**

- Post logs, screenshots, etc

**Things to look for during review**

- [ ] Make sure all references to `phillips` class prefix is using the prefix variable
- [ ] All major areas have a `data-testid` attribute.
- [ ] Document all props with jsdoc comments
- [ ] All strings should be translatable.
- [ ] Unit test should be written and should have a coverage of 90% or higher in all areas.
- [ ] PR should link and close out an existing issue
- [ ] Make sure all commits messages follow convention and are appropriate for the changes

New Components

- [ ] Default story called "Playground" should be created for all new components
- [ ] Create a jsdoc comment that has an Overview section and a link to the Figma design for the component
- [ ] Export the component and its typescript type from the `index.ts` file

[L3-2028]: https://phillipsauctions.atlassian.net/browse/L3-2028?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by davidicus — APPROVED** [review]
This looks ok to me. I guess I misunderstood where we were setting the storybook link. I am not sure of the utility of adding a link to the page the user is already on.

**Comment by scottdickerson** [comment]
remin

> This looks ok to me. I guess I misunderstood where we were setting the storybook link. I am not sure of the utility of adding a link to the page the user is already on.

I think @davidicus is right, now that I see it it doesn't make sense to link back to the current page.  @constantinehuzenko can you remove the storybook links?

File: src/components/Grid/Grid.tsx [hunk 1]
```diff
@@ -13,6 +13,16 @@ export interface GridProps extends React.HTMLAttributes<HTMLElement> {
   element?: React.ElementType<GridProps>;
 }
 
+/**
+ * ## Overview
+ *
+ * A wrapper component for adding a 12-column grid layout.
+ *
+ * [Figma Link](https://www.figma.com/design/xMuOXOAKVt5HC7hgYjF3ot/Components-v2.0?node-id=6226-1330&m=dev)
```
Comment (scottdickerson) [inline src/components/Grid/Grid.tsx:21]: I think this is a more direct link and should be used on GridItem as well.

```suggestion
 * [Figma Link](https://www.figma.com/design/xMuOXOAKVt5HC7hgYjF3ot/Components-v2.0?node-id=7274-9908&m=dev)
```

File: src/components/Input/Input.tsx [hunk 2]
```diff
@@ -95,6 +95,20 @@ export interface InputProps extends Record<string, unknown> {
   warnText?: React.ReactNode;
 }
 
+/**
+ * ## Overview
+ *
+ * A page will usually contain multiple Rows.  The Row component will apply paddings to the contents within it.  Usually a Grid will be rendered within a Row to align to the grid, but other elements are supported.
+ *
+ * [Figma Input Link](https://www.figma.com/design/xMuOXOAKVt5HC7hgYjF3ot/Components-v2.0?node-id=4433-163016&m=dev)
```
Comment (scottdickerson) [inline src/components/Input/Input.tsx:103]: I think this is a more direct link.

```suggestion
 * [Figma Input Link](https://www.figma.com/design/xMuOXOAKVt5HC7hgYjF3ot/Components-v2.0?node-id=5730-11727&m=dev)
```

File: src/components/Link/Link.tsx [hunk 3]
```diff
@@ -33,8 +33,11 @@ export interface LinkProps extends HTMLAttributes<HTMLAnchorElement> {
  *
  * A component that can be used to navigate to different pages or external websites. Renders a standard anchor tag by default.
  *
- * [Figma Link](https://www.figma.com/file/xMuOXOAKVt5HC7hgYjF3ot/Components-v2.0?node-id=5736%3A13364&mode=dev)
+ * [Figma Link](https://www.figma.com/design/xMuOXOAKVt5HC7hgYjF3ot/Components-v2.0?node-id=4612-79026&m=dev)
```
Comment (scottdickerson) [inline src/components/Link/Link.tsx:36]:
```suggestion
 * [Figma Link](https://www.figma.com/design/xMuOXOAKVt5HC7hgYjF3ot/Components-v2.0?node-id=5731-12815&m=dev)
```

File: src/components/LinkBlock/LinkBlock.tsx [hunk 4]
```diff
@@ -15,7 +15,9 @@ export interface LinkBlockProps extends React.HTMLAttributes<HTMLDivElement> {
  *
  * The LinkBlock component is used to render a link with a description underneath it. Usually this is used in a list of links.
  *
- * [Figma Link](https://www.figma.com/file/xMuOXOAKVt5HC7hgYjF3ot/Components-v2.0?type=design&node-id=5709-8035&mode=design&t=CTNssP89Nrnt6Jkw-0)
+ * [Figma Link](https://www.figma.com/design/xMuOXOAKVt5HC7hgYjF3ot/Components-v2.0?node-id=4612-79026&m=dev)
```
Comment (scottdickerson) [inline src/components/LinkBlock/LinkBlock.tsx:18]:
```suggestion
 * [Figma Link](https://www.figma.com/design/xMuOXOAKVt5HC7hgYjF3ot/Components-v2.0?node-id=5731-12815&m=dev)
```

**Review by scottdickerson — CHANGES_REQUESTED** [review]
Looks like SplitPanel didn't get updated to have a storybook link?  I'd use this link:

https://www.figma.com/design/Hp2FyltbOmRxTuw9kSwBAd/EPIC-About-Us?node-id=904-13262&t=k5gTlHC8QkLmwTFs-4

**Comment by scottdickerson** [comment]
I think this is a `docs` change not a `feat` change.  We shouldn't need to publish a new release

**Comment by davidicus** [comment]
> I think this is a `docs` change not a `feat` change. We shouldn't need to publish a new release

good catch

**Review by scottdickerson — APPROVED** [review]

**Pull request merged**